pyproject.toml

```diff
@@ -3,3 +3,28 @@ line-length = 100
 
 [tool.isort]
 line_length = 100
+
+[tool.poetry]
+name = "securedrop-proxy"
+version = "0.1.0"
+description = "Qubes RPC proxy service for the SecureDrop Client"
+authors = ["SecureDrop Team"]
+license = "GPLv3+"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+pyyaml = "^5.4.1"
+requests = "^2.31.0"
+
+[tool.poetry.group.dev.dependencies]
+black = "^23.7.0"
+coverage = "^7.3.0"
+flake8 = "^6.1.0"
+isort = "^5.12.0"
+mypy = "^1.5.1"
+types-PyYAML = "^6.0.12.11"
+types-requests = "^2.31.0.2"
+vcrpy = "^5.1.0"
+bandit = "*"
+safety = "*"
```

Comment:
We previously did not specify this as a test dependency at all, and just `pip install`ed it in the Makefile targets.

I think it's best to enumerate all test dependencies as such. If we want to make sure that we always get the latest version, we can `poetry update` the dependency before each run. That's what the Makefile targets do now.

Reply:
Yeah, upon reflection, I moved this out of the Makefile target and into the CI job at least. IMO it should always be clearly the developer's intent to update.